urbansim/models/dcm.py

```diff
@@ -812,7 +815,13 @@ def predict_from_cfg(cls, choosers, alternatives, cfgname,
         lcm : MNLDiscreteChoiceModel which was used to predict
         """
         logger.debug('start: predict from configuration {}'.format(cfgname))
-        lcm = cls.from_yaml(str_or_buffer=cfgname)
+        try:
+            if cfgname:
+                lcm = cls.from_yaml(str_or_buffer=cfgname)
+            else:
+                lcm = cls.from_yaml(yaml_str=cfg)
+        except:
+            logger.debug('no lcm config loaded')
```

Comment:
I would use `logger.exception` here to so that the error gets logged at the ERROR level and the exception information shows up in the logs. I would also re-raise the exception here so that execution is stopped.